extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -340,7 +340,7 @@ oppia.directive('imageWithRegionsEditor', [
                 // that doesn't overlap with currently existing labels.
                 var newLabel = null;
                 for (var i = 1; i <= labels.length + 1; i++) {
-                  var candidateLabel = 'Region ' + i.toString();
+                  var candidateLabel = 'Region' + i.toString();
```

Comment:
I did a little investigation. The default name change is due to this:

https://github.com/Arunabh98/oppia/blame/f45ea587362b39107b64d2ec230ced778a5177d9/extensions/objects/templates/ImageWithRegionsEditor.js#L343

and I agree that your fix looks good! Though at some point we should figure out why spaces were disallowed.